internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -4590,3 +4621,36 @@ func retrySystemNodePoolCreation(ctx context.Context, client *agentpools.AgentPo
 
 	return err
 }
+
+func expandKubernetesClusterUpgradeOverrideSetting(input []interface{}) *managedclusters.ClusterUpgradeSettings {
+	if len(input) == 0 || input[0] == nil {
+		return &managedclusters.ClusterUpgradeSettings{
+			OverrideSettings: &managedclusters.UpgradeOverrideSettings{
+				// The forceUpgrade field controls the overrideUpgradeSetting feature.
+				// If overrideUpgradeSetting is not specified, set forceUpgrade to false to disable the feature.
+				// Not passing this field to the API will not disable the feature.
```

Comment:
Just clarifying here, are you saying that if I created a fresh cluster and omitted the entire `UpgradeOverrideSettings` block in the payload, the API will return this block with `ForceUpgrade` set to `true`?